### ISSUE TITLE
A11y – Fix internet-header post-meta-navigation aria-hidden on name

### ISSUE THREAD

**Issue by imagoiq** [swisspost/design-system#2687]
Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute"

Reported by cypress-axe (4 nodes)

![Cypress_GMVeCDlS6J](https://github.com/swisspost/design-system/assets/12294151/6ce84870-8061-4ad0-ac83-0e9bf31685f7)
